scripts/opx_build

```diff
@@ -53,6 +53,10 @@ repos=( opx-logging
 	opx-tmpctl
 	)
 
+if [ ! -z "$1" ] && [ "$1" != "all" ]; then
```

Comment:
Agree, not needed. Just done to be similar with "make all". Can be removed.
@johntconklin What is your suggestion here?